cookbooks/travis_build_environment/recipes/jq.rb

```diff
@@ -22,12 +22,22 @@
 # OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
 # WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
-arch = node['kernel']['machine'] =~ /x86_64/ ? '64' : '32'
+arch = node['kernel']['machine']
+if arch == 'ppc64le'
```

Comment:
Likewise here, this condition should be part of each resource as `not_if` / `only_if` rather than an outer guard.